test/test_lldp.cpp

```diff
@@ -59,3 +59,37 @@ TEST_F(LldpTest, LldpInitTest)
    EXPECT_EQ(appdata.call_counters.write_calls, 0);
    EXPECT_EQ(mock_os_data.eth_send_count, 1);
 }
+
+TEST_F(LldpTest, LldpGenerateAliasName)
+{
+   char alias[100];
+   int err; 
+
+   err =  pf_lldp_generate_alias_name(NULL, "dut", alias , 96);
+   EXPECT_EQ(err, -1);
+   
+   err =  pf_lldp_generate_alias_name("port-001", NULL, alias , 96);
+   EXPECT_EQ(err, -1);
+
+   err =  pf_lldp_generate_alias_name("port-001", "dut", NULL , 96);
+   EXPECT_EQ(err, -1);
+
+
+   err =  pf_lldp_generate_alias_name("port-001", "dut", alias, 13);
+   EXPECT_EQ(err, 0);
+   printf("alias :%s\n",alias);
```

Comment:
Unit tests should preferably not clutter the output. See the TEST_TRACE macro if the output is useful for debugging the tests.

Reply:
Good point, the printf was left by mistake. I have pushed a version where it is removed

Reply:
The 'printf' does not look removed. Am I looking at the wrong version?

Reply:
No, you are correct. I failed to remove the printf in the version i pushed. I however pushed a new update which fixes this.